spec/index.adoc

```diff
@@ -2849,6 +2849,9 @@ The controllers of a canister can obtain information about the canister.
 
 The `Memory_size` is the (in this specification underspecified) total size of storage in bytes.
 
+The `freezing_threshold_cycles` is the freezing threshold in cycles of the canister, which is computed based on its current memory footprint, storage cost, memory and compute allocation, and current `freezing_threshold` setting.
```

Comment:
I think we should explain the meaning of this value, it is not at all clear from the description. For example,

> The `freezing_threshold_in_cycles` is the current estimate of the smallest amount of cycles the canister can have without entering the frozen state. The system uses the `freezing_threshold` setting to compute this field.